packages/@nucleus/tests/dummy/app/templates/components/nucleus-button/demo-2.hbs

```diff
@@ -0,0 +1,10 @@
+{{#docs-demo as |demo|}}
+  {{#demo.example name="nucleus-button-group.hbs"}}
+  {{#nucleus-button-group  as |button|}}
```

Comment:
Kindly consider the below syntax. nucleus-button-group as buttons makes it more readable.

`
{{#nucleus-button-group  as |buttons|}}
  {{#buttons.button}}Button1{{/buttons.button}}
  {{#buttons.button}}Button2{{/buttons.button}}
{{/nucleus-button-group}}
`